privacyidea/api/lib/postpolicy.py

```diff
@@ -503,7 +504,7 @@ def get_webui_settings(request, response):
             realm=realm,
             client=client
         )
-        timeout_action = policy_object.get_policies(
+        timeout_action_pol = policy_object.get_policies(
```

Comment:
I would recommend using ``get_action_values``, so you can directly receive the list of timeout_actions. (see logout_time_pol).

This way you later do not need to do:

       timeout_action = timeout_action_pol[0]['action']['timeout_action']

but only:

       timeout_action = timeout_action_pol[0] 

This this will be a list of action values.

If using ``unique = True`` in ``get_action_values``you will get an exception if there are contradicting policies. As this is also done with logout_time_pol at the moment, we probably should also do this with logout_action.